### DISCUSSION TITLE
Issue with Displaying Submissions on Mosparo Dashboard

### DISCUSSION THREAD

**Discussion by diegodmzr** [mosparo/mosparo#177] [Problems & Issues]
Hello Mosparo Team,

I hope this message finds you well. I'm reaching out regarding an issue I'm experiencing with the display of submissions on the dashboard of my self-hosted Mosparo project running on Ubuntu. I'm using Vue.js for the login page, and I've converted the code provided in the documentation to Vue.

Despite the overall functionality working correctly, no submissions are being displayed on the dashboard after captcha validation. I've followed the documentation closely for the integration, but the issue persists.

Could you please assist me in identifying the cause of this malfunction and guide me in resolving it promptly?

Thank you in advance for your assistance.

Best regards,

Screenshot :
![Capture d’écran du 2024-01-19 10-45-10](https://github.com/mosparo/mosparo/assets/157109129/602769d9-fa42-498d-9ede-7b24be5a4588)
![image](https://github.com/mosparo/mosparo/assets/157109129/b5a7fa3a-4dd6-4640-b0e0-12bfa4212a28)
![image](https://github.com/mosparo/mosparo/assets/157109129/63bbbf6e-0869-40d9-84db-f8209286ed09)
![image](https://github.com/mosparo/mosparo/assets/157109129/a40b4b15-7be2-44de-af4b-3242198e12ff)

**Comment by zepich** [comment]
Hi @diegodmzr 

Thank you very much for your message and sharing your code.

The submission is only visible in the dashboard and the list of submissions when the submission was verified in the backend of your website.

To make sure that the user cannot manipulate the form submission, it is required to execute both mosparo processes: the frontend validation by mosparo (by clicking the checkbox/with the code you have in the third screenshot) and the backend verification in the backend of your website before you process the form data (for example, before you send the email or store the data in the database).

Do you also have the backend verification? Can you show that part of the code, too?

Documentation: https://documentation.mosparo.io/docs/integration/custom#performing-verification

Kind regards,

zepich

**Reply by diegodmzr** [reply]
Hello Mosapro Team,

I hope this message finds you well.

I wanted to express my gratitude for your assistance regarding the CORS issue. I've implemented the suggested changes, but unfortunately, I'm still encountering problems.

The CORS error persists, and I would appreciate any further guidance or recommendations you may have.

Thank you once again for your support.

Best regards,
